src/Infrastructure.EntityFramework/Repositories/Queries/UserCipherDetailsQuery.cs

```diff
@@ -78,6 +88,71 @@ public virtual IQueryable<CipherDetails> Run(DatabaseContext dbContext)
         return union;
     }
 
+    private IQueryable<CipherDetails> Run_V2(DatabaseContext dbContext)
```

Comment:
💭 This is a lot of EF code ... will be valuable to debug its resulting SQL. I also support EF calling stored procedures to get known and measured performance.